### PR TITLE
MBS-9935: Indicate what to do with annotation reports

### PR DESCRIPTION
**Pull request by reosarevok**
https://tickets.metabrainz.org/browse/MBS-9935

To be reviewed and merged after https://github.com/metabrainz/musicbrainz-server/pull/934

### CODE REVIEW

File: root/report/AnnotationsArtists.js [hunk 1]
```diff
@@ -32,9 +32,17 @@ const AnnotationsArtists = ({
       <li>
         {l('This report lists artists with annotations.')}
       </li>
+      <li>
+        {l(`If you see something in these annotations that can be represented
+            with a relationship instead, please add a relationship and remove
+            that part of the annotation. If something is marked as
+            "sub-optimal", consider checking if a better way to store that
```
Comment (yvanzo) [inline root/report/AnnotationsArtists.js]: `“”` should probably be preferred so as to follow [STYLE guidelines](https://musicbrainz.org/doc/Style/Miscellaneous#Use_of_special_characters).

File: root/report/AnnotationsArtists.js [hunk 2]
```diff
@@ -32,9 +32,17 @@ const AnnotationsArtists = ({
...
+            data has been added in the meantime.`)}
+      </li>
       <li>
         {texp.l('Total artists found: {count}',
                 {count: pager.total_entries})}
+
```
Comment (yvanzo) [inline root/report/AnnotationsArtists.js]: unexpected change

**Comment by mwiencek** [comment]
These are all identical, right? Can we move it to ./root/report/constants or somewhere?

File: root/report/AnnotationsArtists.js (hunk 2 above)
Comment (yvanzo) [reply root/report/AnnotationsArtists.js]: resolved in https://github.com/metabrainz/musicbrainz-server/pull/947/commits/69a313ae52c9763cf3117a9e45c71a1a2918448d

File: root/report/AnnotationsArtists.js (hunk 1 above)
Comment (yvanzo) [reply root/report/AnnotationsArtists.js]: resolved in https://github.com/metabrainz/musicbrainz-server/pull/947/commits/69a313ae52c9763cf3117a9e45c71a1a2918448d

**Pull request merged**